api/swagger-spec/v1beta1.json

```diff
@@ -2289,11 +2289,11 @@
      },
      "metadata": {
       "$ref": "v1.ObjectMeta",
-      "description": "Standard object metadata. More info: http://releases.k8s.io/HEAD/docs/devel/api-conventions.md#metadata"
+      "description": "Standard object metadata. More info: http://releases.k8s.io/v1.1.0/docs/devel/api-conventions.md#metadata"
```

Comment:
This is for 1.1 release branch on github.
Before moving it to gh-pages, we will replace releases.k8s.io/v1.1.0 by k8s.io/1.1.